scripts/parachain/local.sh

```diff
@@ -27,7 +27,7 @@ cargo build --features parachain --manifest-path ../../node/Cargo.toml --release
 ../release/zeitgeist export-genesis-wasm --chain $CHAIN > para-wasm
 
 ./target/release/polkadot build-spec --chain rococo-local --disable-default-bootnode > rococo-local-plain.json
-echo "s/\"paras\": \[\]/\"paras\": [[9123, { \"genesis_head\": \"$(cat para-genesis)\", \"validation_code\": \"$(cat para-wasm)\", \"parachain\": true }]]/" > sed.txt
+echo "s/\"paras\": \[\]/\"paras\": [[$PARACHAIN_ID, { \"genesis_head\": \"$(cat para-genesis)\", \"validation_code\": \"$(cat para-wasm)\", \"parachain\": true }]]/" > sed.txt
```

Comment:
In combination with `set -x`, this spams the console with the wasm blob.

One solution:
```suggestion
set +x
SET_MSG="+ echo '\"s/\"paras\": \[\]/\"paras\": [[$PARACHAIN_ID, { \"genesis_head\": \"$(cat para-genesis)\", \"validation_code\": \"--snip--\", \"parachain\": true }]]/\"' > sed.txt"
echo "$SET_MSG"
echo "s/\"paras\": \[\]/\"paras\": [[$PARACHAIN_ID, { \"genesis_head\": \"$(cat para-genesis)\", \"validation_code\": \"$(cat para-wasm)\", \"parachain\": true }]]/" > sed.txt
set -x
```